CONTRIBUTING.md

```diff
@@ -100,7 +100,11 @@ To be considered popular enough to be within our scope, a brand must be in exist
     - [npm](https://www.npmjs.com): 100k weekly downloads,
     - [JSDelivr](https://www.jsdelivr.com): 1m daily or 35m monthly requests,
     - [crates.io](https://crates.io): 100k weekly downloads,
-    - [PyPi Stats](https://pypistats.org): 100k weekly downloads, or,
+    - [PyPi Stats](https://pypistats.org): 100k weekly downloads,
+    - [pkg.go.dev](https://pkg.go.dev): 100k importers,
+    - [Homebrew Formulae](https://formulae.brew.sh): 100k weekly downloads,
```

Comment:
These need to be updated to be a) realistic for the package manager and b) actually tracked by the package manager! Homebrew doesn't have a weekly downloads count, and 100k is very high for a weekly count anyway. We can and should reference the statistics from Homebrew for packages/apps/products already in our collection to determine the right number.

| Package | Last 30 days | Last 90 days | Last 365 days |
| --- | --- | --- | --- |
| `deno` | 13k | 40k | 145k |
| `go` | 87k | 261k | 1m |
| `1password` (cask) | 3k | 9k | 39k |
| `firefox` (cask) | 11k | 35k | 140k |
| `inkscape` (cask) | 2k | 6k | 26k |
| `obsidian` (cask) | 5k | 15k | 58k |
| `raycast` (cask) | 7k | 21k | 70k |
| `spotify` (cask) | 6k | 21k | 83k |

Reply:
Oh, dang it. I just defaulted to the same minimum as everything else.